lineapy/plugins/base_pipeline_writer.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         self.artifact_collection = artifact_collection
         self.keep_lineapy_save = keep_lineapy_save
         self.pipeline_name = slugify(pipeline_name)
-        self.output_dir = Path(output_dir)
+        self.output_dir = Path(output_dir).expanduser()
```

Comment:
any reason this is here and not in api.py?

Reply:
This is where it first becomes a path object. in api.py and here in parameters it is still a string.